### DISCUSSION TITLE
Welcome to JFreeChart Discussions!

### DISCUSSION THREAD

**Discussion by jfree** [jfree/jfreechart#272] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tracylynne99** [comment]
Tracy here! I've been using JFreeChart as part of my job since 2005 or so. Extended it to create a custom gantt chart for our users.  I also try to help out with maintaining the JFC baseline as much as possible!

**Reply by jfree** [reply]
Welcome Tracy, thanks for all your help so far!

**Reply by mikalziane** [reply]
Hello there.

I have tried several times to buy the developer's guide, today and some days ago, but a message keeps popping saying to try again in ... a few minutes!

**Reply by trashgod** [reply]
@mikalziane: See also issue #253.
